.azuredevops/modulePipelines/ms.aad.domainservices.yml

```diff
@@ -28,7 +28,7 @@ trigger:
       - '/**/*.md'
 
 variables:
-  - template: '../../global.variables.yml'
+  - template: '../../settings.yml'
```

Comment:
Are we sure we want to rename this file? To me its values look more like variables than settings, but if you all agree it makes more sense to call it settings then it's fine.

Reply:
I think one could make a case either way as there are both settings (e.g., telemetry) as well as classic variables (e.g., default RG) in the file.. And I'm fine either way.
Maybe we can get some more opinions on that 😊

Reply:
@MariusStorhaug initially provided the idea here https://github.com/Azure/ResourceModules/issues/1363#issuecomment-1166508084 and I think it's a good name to have hence suggested it. It is kind of that centralized place for config which is typically referred to as settings :P